client/components/data/store-connection/index.jsx

```diff
@@ -58,17 +58,22 @@ const StoreConnection = React.createClass( {
 	},
 
 	render() {
+		const props = {
+			...this.props,
```

Comment:
> increase the chance of regression.

yeah I'll be reviewing that before pushing this out. like I said, I have this component targeted for another PR. thanks for the catch!